nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/GetS3ObjectMetadata.java

```diff
@@ -94,12 +80,23 @@ public class GetS3ObjectMetadata extends AbstractS3Processor {
             .required(true)
             .build();
 
+    public static final PropertyDescriptor ATTRIBUTE_INCLUDE_PATTERN = new PropertyDescriptor.Builder()
+            .name("Attribute Include Pattern")
```

Comment:
For consistency, naming this `Metadata Include Pattern` would help indicate that it used to match against to the S3 Metadata Key, as opposed to the FlowFile Attribute. Or to cover both bases, `Metadata Attribute Include Pattern`.